openmrs/src/main/java/org/motechproject/openmrs/tasks/builder/ActionBuilder.java

```diff
@@ -49,8 +50,10 @@ public ActionBuilder(OpenMRSConfigService configService) {
             actions.add(buildCreateEncounterAction(configName));
             actions.add(buildCreatePatientAction(configName));
             actions.add(buildUpdatePatientAction(configName));
-            actions.add(buildCreateProgramEnrollmentAction(configName));
-            actions.add(buildChangeStateOfProgramEnrollmentAction(configName));
+            if(config.getOpenMrsVersion().equals(OPENMRS_V1_12)) {
```

Comment:
if you compare two strings, always start with raw one, it prevents NPE. Or you can use always StringUtils. 

Apart from that, I'm consider how to handle this, because I suppose that in the future maybe there will be another openMRS version like 1.11,  2.0 and so on. So maybe here the best solution is to compare with 1.9 and only if equals to 1.9 we do not add this actions. Moreover it will be more readable if you move this condition into private method. Lastly, please move this few lines at the end of adding actions code